.vscode/launch.json

```diff
@@ -54,7 +54,7 @@
 			"sourceMaps": true,
 			// outFiles is used for locating generated JavaScript files, see https://code.visualstudio.com/docs/nodejs/nodejs-debugging#_source-maps
 			"outFiles": [
-				"${workspaceFolder}/dist/**/*.js"
+				"${workspaceFolder}/out/**/*.js"
```

Comment:
Is this change tangential to the PR?

Reply:
Yes, don't worry about it, I'll fix it when merging the webpack PR.